lib/manager/gradle-wrapper/extract.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  extractPackageFile,
+};
+
+function extractPackageFile(fileContent) {
+  logger.debug('gradle-wrapper.extractPackageFile()');
+  const lines = fileContent.split('\n');
+
+  let lineNumber = 0;
+  for (const line of lines) {
+    const match = line.match(
+      /^distributionUrl=https\\:\/\/services.gradle.org\/distributions\/gradle-((\d|\.)+)-(bin|all)\.zip\s*$/
+    );
+    if (match) {
+      const dependency = {
+        purl: 'pkg:gradle',
+        depType: 'gradle-wrapper',
+        depName: 'gradle',
+        currentValue: match[1],
+        lineNumber,
+        versionScheme: 'semver',
+        /* gradle specifics ... */
+        type: match[3],
+      };
+
+      let shaLineNumber = 0;
+      for (const shaLine of lines) {
+        const shaMatch = shaLine.match(/^distributionSha256Sum=((\w){64}).*$/);
+        if (shaMatch) {
+          dependency.shaCurrentValue = shaMatch[1];
```

Comment:
This seems to indicate that there's only one dependency line and only one sha line?